financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/SharedPartnerAuth.kt

```diff
@@ -233,37 +215,6 @@ private fun SharedPartnerAuthBody(
     }
 }
 
-/**
- * Wrapper for the content of the partner auth screen, that based on the [inModal] parameter
- * will render the content in a modal or in a full screen.
- */
-@Composable
-private fun SharedPartnerAuthContentWrapper(
```

Comment:
The `SharedPartnerAuthContentWrapper` is redundant now, as the `AuthBody` never displays a top app bar now.